src/modules/users/entities/user.entity.ts

```diff
@@ -19,4 +26,10 @@ export class User {
 
   @Column()
   password: string;
+
+  @ManyToOne(() => Role, (role) => role.users, {
+    eager: true,
+  })
+  @JoinColumn({ name: 'role_id' })
```

Comment:
We store relation id on the user side, so we don't have to join role table every time we querying users.
Use decorator _RelationId_ for retrieving role id as user entity field